pallets/gear/src/schedule.rs

```diff
@@ -512,7 +515,7 @@ impl Default for Limits {
             // 4k function pointers (This is in count not bytes).
             table_size: 4096,
             br_table_size: 256,
-            subject_len: 32,
+            subject_len: 1024,
```

Comment:
@ukint-vs  why we doesn't use it anywhere in `gr_random` syscall?

Reply:
@ukint-vs may be we should use old small value `32` and add restriction in `gr_random` ? Then this benchmarks has no sence.

Reply:
> @ukint-vs why we doesn't use it anywhere in `gr_random` syscall?

I added a comment above.